Character.cpp

```diff
@@ -113,7 +113,9 @@ void Character::getCharacterReward(int& initial, int& current)
 {
     if( current < initial )
     {
-        current = initial + (initial * 0.1);
+        current = initial;
+        current *= 0.1;
+        initial = current;
     }
     
```

Comment:
This is incorrect. 
```
        When you defeat another Character: 
            a) your stats are restored to their initial value if they are lower than it.
            b) your stats are boosted 10%
            c) the initial value of your stats is updated to reflect this boosted stat for the next time you defeat another character.
```
you only do `a` when the `if` condition is `true`
you always do `b`
you always do `c`.
The logic in this function needs to implement that spec.
- [ ] revise